sentry-core/src/main/java/io/sentry/core/SentryClient.java

```diff
@@ -117,6 +117,34 @@ public SentryId captureEvent(SentryEvent event, @Nullable Scope scope) {
     return event.getEventId();
   }
 
+  private SentryEvent executeBeforeSend(SentryEvent event) {
+    SentryOptions.BeforeSendCallback beforeSend = options.getBeforeSend();
+    if (beforeSend != null) {
+      try {
+        event = beforeSend.execute(event);
+      } catch (Exception e) {
+        logIfNotNull(
+            options.getLogger(),
+            SentryLevel.ERROR,
+            "The BeforeSend callback threw an exception. It will be added as breadcrumb and continue.",
+            e);
+
+        Breadcrumb breadcrumb = new Breadcrumb();
+        breadcrumb.setMessage("BeforeSend callback failed.");
+        breadcrumb.setCategory("SentryClient");
```

Comment:
`category` is usually all lower case. Also, the[ docs has more info](https://docs.sentry.io/development/sdk-dev/event-payloads/breadcrumbs/):

> Optional. A dotted string indicating what the crumb is or from where it comes. Typically it is a module name or a descriptive string. For instance, ui.click could be used to indicate that a click happened in the UI or flask could be used to indicate that the event originated in the Flask framework.

Maybe something like?
```suggestion
        breadcrumb.setCategory("callback");
```